action.yml

```diff
@@ -17,8 +17,8 @@ inputs:
     description: 'Also look for tags to delete'
     required: false
     default: false
-  include_prefixes:
-    description: 'List of branch prefixes to be deleted'
+  extra_protected_branch_prefix:
+    description: 'Comma separated list of branch prefixes to exclude from deletion'
```

Comment:
Should we call this `exclude_prefix`? or even better `exclude_regexp` and support regexp?

Reply:
a comma separated list of regexes, correct?

Reply:
we could do it with an regex OR operator... `^(part1|part2)$`

Reply:
Hmm OK I will change it then.